src/main/java/com/google/cloud/tools/gradle/appengine/core/AppEngineCorePluginConfiguration.java

```diff
@@ -126,7 +127,11 @@ private void createDownloadCloudSdkTask() {
             downloadCloudSdkTask -> {
               downloadCloudSdkTask.setGroup(taskGroup);
               downloadCloudSdkTask.setDescription("Download the Cloud SDK");
-              downloadCloudSdkTask.requiresAppEngineJava(requiresAppEngineJava);
+
+              // make sure we download our required components
+              if (requiresAppEngineJava) {
+                downloadCloudSdkTask.requiresComponent(SdkComponent.APP_ENGINE_JAVA);
```

Comment:
There's no prod code that requests to install `SdkComponent.BETA`. Is the API change necessary?

Reply:
Yeah, I made these changes earlier to handle pre-isntalling BETA but then I liked them so I left it in.